app/src/main/java/com/rockthevote/grommet/data/db/model/ContactMethod.java

```diff
@@ -1,60 +0,0 @@
-package com.rockthevote.grommet.data.db.model;
-
-import android.os.Parcelable;
-import android.provider.BaseColumns;
-import androidx.annotation.Nullable;
-
-public abstract class ContactMethod implements Parcelable, BaseColumns {
-
-    public enum Type {
-        PHONE("phone"),
-        ASSISTANT_PHONE("assistant_phone"),
```

Comment:
If I understand this correctly (from the comment in ApiContactMethod), the only point in `assistant_phone` was due to a database constraint, and that it would otherwise be specified as just `phone` anywhere else. Let me know if there's another use.